trio/_tests/test_windows_pipes.py

```diff
@@ -1,37 +1,47 @@
+from __future__ import annotations
+
 import sys
-from typing import Any, Tuple
+from typing import TYPE_CHECKING, Any
 
 import pytest
 
 from .. import _core
 from ..testing import check_one_way_stream, wait_all_tasks_blocked
 
+# Mark all the tests in this file as being windows-only
+pytestmark = pytest.mark.skipif(sys.platform != "win32", reason="windows only")
+
+assert (
+    sys.platform == "win32" or not TYPE_CHECKING
+)  # Skip type checking when not on Windows
```

Comment:
nitpick: I think comment above might look nicer for this.

Reply:
It was originally, but then when black formatted it it moved it below and I didn't notice. Fixed in a92b9c8